Common/Sources/Localization/cs.lproj/Localizable.strings

```diff
@@ -21,39 +21,40 @@
 "remove" = "Odebrat";
 "cancelButtonLabel" = "Zrušit";
 "doneButtonLabel" = "Hotovo";
+"edit" = "Upravit";
 "try_again_button_label" = "Zkusit znovu";
 "search_placeholder_label" = "Vyhledat";
 "unexpected_error_message" = "Něco se pokazilo, zkuste to prosím znovu";
 "network_connection_error_title" = "Problém s připojením";
-"unexpected_title" = "Neočkávaná chyba";
+"unexpected_title" = "Neočekávaná chyba";
 "network_connection_error_message" = "Máme problém se zpracováním požadavku. Zkontrolujte prosím své připojení.";
 "endingIn" = "končí v ";
 "to" = "pro ";
 "no_country_available_error_message" = "Není k dispozici žádá země";
 "no_transfer_method_available_error_message" = "Pro %1$@ a %2$@ není k dispozici žádný způsob převodu";
-"no_currency_available_error_message" = "Pro zemi %1$@ není k dispozici žádná měna";
+"no_currency_available_error_message" = "Pro zemi  není k dispozici žádná měna%1$@";
 "emptyStateAddTransferMethod" = "Chcete-li začít, přidejte způsob převodu.";
-"mobileAddTransferMethodHeader" = "Přidejte způsob převodu";
+"mobileAddTransferMethodHeader" = "Přidat způsob převodu";
 "mobileCountryCurrencyLabel" = "MÍSTO A MĚNA";
 "mobileTransferMethodLabel" = "ZPŮSOB PŘEVODU";
 "mobileCountryRegion" = "Země / oblast";
 "mobileCurrencyLabel" = "Měna";
-"mobileAreYouSure" = "Jste si jistí?";
-"fee_flat_formatter" = "Poplatek %1$@%2$@ ";
-"fee_percent_only_min_formatter" = "Poplatek %1$@%% (min.:%2$@%3$@)";
-"fee_percent_only_max_formatter" = "Poplatek %1$@%% (max.:%2$@%3$@)";
-"fee_percent_formatter" = "Poplatek %1$@%% (min.:%2$@%3$@, max.:%2$@%4$@)";
-"fee_mix_no_min_and_max_formatter" = "Poplatek %1$@%2$@ + %3$@%%";
-"fee_mix_only_min_formatter" = "Poplatek %1$@%2$@ + %3$@%% (min.:%1$@%4$@)";
-"fee_mix_only_max_formatter" = "Poplatek %1$@%2$@ + %3$@%% (max.:%1$@%4$@)";
-"fee_mix_formatter" = "Poplatek %1$@%2$@ + %3$@%% (min.:%1$@%4$@, max.:%1$@%5$@)";
-"fee_percent_no_min_and_max_formatter" = "Poplatek %1$@%%";
+"mobileAreYouSure" = "Odebrat tento způsob převodu?";
+"fee_flat_formatter" = "Poplatek %1$@%2$@";
+"fee_percent_only_min_formatter" = "Poplatek %1$@%% (min.:%2$@%3$@";
```

Comment:
I think these are messed up? Its missing a closing bracket. @malaw-hw @SaiNarendran can you confirm?

Reply:
Updated.